sdk/src/main/java/com/yellowpineapple/wakup/sdk/activities/ParentActivity.java

```diff
@@ -129,6 +134,17 @@ public void setSubtitle(final CharSequence subtitle) {
         }
     }
 
```

Comment:
I think that with the new Toolbar implementation, overriding `setTitle` method will not be required anymore.
Alse, `setSubtitle` method should be removed